tests/Unit/Framework/Pypp.hpp

```diff
@@ -26,58 +26,40 @@
 #include "Utilities/TypeTraits/IsA.hpp"
 #include "Utilities/TypeTraits/IsStdArray.hpp"
 
-/// \ingroup TestingFrameworkGroup
-/// Contains all functions for calling python from C++
+/*!
+ * \ingroup TestingFrameworkGroup
+ * \brief Contains all functions for calling python from C++
+ */
 namespace pypp {
 namespace detail {
-template <typename R>
-R call_work(PyObject* python_module, PyObject* func, PyObject* args) {
-  PyObject* value = PyObject_CallObject(func, args);
-  Py_DECREF(args);  // NOLINT
-  if (value == nullptr) {
-    Py_DECREF(func);           // NOLINT
-    Py_DECREF(python_module);  // NOLINT
-    PyErr_Print();
-    throw std::runtime_error{"Function returned null"};
-  }
+template <typename T, typename ConversionClass>
+struct PackedTypeMatches : std::false_type {};
```

Comment:
is it viable to just create a type alias for `std::is_same_v<>` (i.e. can you assume that all of the conversion classes have the desired type alias)?

Reply:
Yes, that's part of the required interface so it should work. I'll give it a shot and report back if I figured out a reason it doesn't :)